### DISCUSSION TITLE
只用Android设备的一些方案思路，求教

### DISCUSSION THREAD

**Discussion by 6078HITHEDAY** [MaaAssistantArknights/MaaAssistantArknights#10590] [Q&A]
# 构建Android应用
- MAA使用adb自动化，使用**shizuku调用adb**，或root。推荐**shizuku**配置简单
- 使用无障碍实现自动化，估计要重写↔
- 问题：怎么构建¿
# 使用Linux软件包，结合**Termux**
- 来个shizuku ![Screenshot_20240912-235152.png](https://github.com/user-attachments/assets/d738b55f-f9da-4931-8c8b-25ac2d138fba)

- 问题：没试过，感觉太烦，还是adb。

**Comment by Rbqwow** [comment]
你可能需要 https://github.com/AegirTech/ArkLights

**Reply by 6078HITHEDAY** [reply]
# 感觉这个软件有些无语，开屏看到广告，与全部授权(有没有用的都要)否则不让你用，最关键的地方还是看图![Screenshot_20240914-001159.png](https://github.com/user-attachments/assets/bf5f5c9c-9dd8-43e6-b34e-6568dfc1035f)
# 还是用自动任务吧![Screenshot_20240914-001415_EDIT_1726244087871.png](https://github.com/user-attachments/assets/bb05e58e-bf24-450d-936b-e29c0e7fbc32)
